src/edu/washington/escience/myria/io/AmazonS3Source.java

```diff
@@ -69,7 +70,7 @@ public AmazonS3Client getS3Client() {
     if (s3Client == null) {
       clientConfig = new ClientConfiguration();
       clientConfig.setMaxErrorRetry(3);
-      s3Client = new AmazonS3Client(new DefaultAWSCredentialsProviderChain());
+      s3Client = new AmazonS3Client(new AnonymousAWSCredentials());
```

Comment:
I couldn't figure this one out, but I asked @senderista about it in https://github.com/uwescience/myria/pull/864.